### ISSUE TITLE
[Secondary Toolbar]: After applying high contrast mode the selected controls text is not visible.

### ISSUE THREAD

**Issue by seahawksq** [mozilla/pdf.js#14649]
After applying high contrast mode, the selected controls text is not visible.
![image](https://user-images.githubusercontent.com/25859586/157341845-ddc58ce0-edc3-4ea4-8c54-07b69349519c.png)
![image](https://user-images.githubusercontent.com/25859586/157341923-c2741dc7-a0ed-4e55-aa42-eff75d377fd1.png)

Configuration:
- Web browser and its version: Chrome Version 97.0.4692.71 (Official Build) (64-bit) /Edge Version 97.0.1072.69 (Official build) (64-bit)
- Operating system and its version: Operating system and its version: 21H2(OS build 22000.376)
- PDF.js version: 2.13.107
- Is a browser extension: No

Steps to reproduce the problem:
1. Turning on high contrast mode in operating system.
2. Navigate to https://mozilla.github.io/pdf.js/web/viewer.html
3. Open the secondary Toolbar.

What is the expected behavior? (add screenshot)
The selected controls text should be visible.

What went wrong? (add screenshot)
The selected controls text is not visible.

**Comment by Snuffleupagus** [comment]
Given https://github.com/mozilla/pdf.js/issues/14649#issuecomment-1169020381, it really sounds like this should be reported in the Chromium bug-tracker instead.
